cmp/grid/Grid.js

```diff
@@ -534,8 +534,9 @@ class GridLocalModel extends HoistModel {
         return {
             track: () => model.sizingMode,
             run: () => {
-                if (model.autosizeOptions.mode !== GridAutosizeMode.ON_SIZING_MODE_CHANGE) return;
-                model.autosizeAsync({showMask: true});
+                if (model.autosizeIsManaged || model.autosizeOptions.mode === GridAutosizeMode.ON_SIZING_MODE_CHANGE) {
```

Comment:
This logic is wrong -- I think it should go ahead and do the resize, if its either 'ON_SIZING_MODE_CHANGE' or 'MANAGED'.   This is the user change we want to be pretty aggressive, and work globally.